src/api.ts

```diff
@@ -35,6 +37,32 @@ export const bas = {
         return ActionsController.getAction(actionId);
     },
 
+    async getParameter(parameterName: string) : Promise<string | null> {
+        const onBasEnv = "env.WS_BASE_URL";
```

Comment:
`"env.WS_BASE_URL";` to constants instead of magic string value?

Reply:
ignore this, see next commt

Reply:
This is constant.